src/pages/Workspaces/Workspaces.test.tsx

```diff
@@ -12,6 +16,12 @@ test('that the Workspaces page header is displayed', () => {
   expect(screen.getByTestId('page-header')).toHaveTextContent('Workspaces');
 });
 
+test('that the WorkspacesGettingStartedCard component is rendered', () => {
+  render(<Workspaces />);
+
+  expect(screen.getByTestId('workspaces-getting-started-card')).toBeTruthy();
```

Comment:
I'm a fan to use the `.toBeInTheDocument()` vs .toBeTruthy.  Just makes it very clear what you are trying to test (aka that it is available ) very clear.